src/cljc/rems/common/catalogue_util.cljc

```diff
@@ -1,9 +1,33 @@
 (ns rems.common.catalogue-util
-  (:require [clojure.string :as str]))
+  (:require [clojure.string :as str]
+            [rems.text :refer [localized]]))
 
-(defn urn? [resid]
+(defn- urn? [resid]
   (and resid (str/starts-with? resid "urn:")))
 
-(defn urn-catalogue-item-link [{:keys [resid]} {:keys [urn-organization]}]
+(defn- urn-catalogue-item-url [resid {:keys [urn-organization]}]
   (when (urn? resid)
     (str (or urn-organization "http://urn.fi/") resid)))
+
+;; EGA catalogue items (i.e. datasets) look like EGAD00001006673 and could be linked to e.g. https://ega-archive.org/datasets/EGAD00001006673
+
+(defn- ega-dataset? [resid]
+  (and resid (str/starts-with? resid "EGAD")))
+
+(defn- ega-catalogue-item-url [resid {:keys [enable-ega ega-organization]}]
+  (when (and enable-ega (ega-dataset? resid))
```

Comment:
As far as I'm concerned this could be an unconditional feature. I'm fine with the feature flag too.

Reply:
URN was a bit clearer with the link style values but the values EGA wants are `"EGAD..."` and that might conflict with someone. We'll have the feature flag anyway.

Reply:
Ok!